docs/reference-config-guide-v2.md

```diff
@@ -299,6 +299,22 @@ fieldsToOmit:
 
 The default value of `defaultOmitRef` is a built-in list  `cluster-compare-built-in` and can still be referenced even if the `defaultOmitRef` is set.
 
+
+#### constructing an item from other items
+
+We allow for an item to be included as a sub-item of other items:
+
+```yaml
+fieldsToOmit:
+   defaultOmitRef: default
+   items:
+    itemA:
+      - pathToKey: spec.selector.matchLabels.k8s-app
+    default:
+      - include: itemA
+      - pathToKey: spec.template.metadata.labels.k8s-app
+```
```

Comment:
Maybe a more concrete example?  Something like we intend to use in the RAN reference; Like having a "common" that excludes a couple annotations, and then a "default" that also excludes "status"?